README.md

```diff
@@ -1,49 +1,87 @@
-# netbox-operator
+# NetBox Operator
 
 **Disclaimer:** This project is currently under development and may change rapidly, including breaking changes. Use with caution in production environments.
 
 NetBox Operator extends the Kubernetes API by allowing users to manage NetBox resources – such as IP addresses and prefixes – directly through Kubernetes. This integration brings Kubernetes-native features like reconciliation, ensuring that network configurations are maintained automatically, thereby improving both efficiency and reliability.
 
-<div align=center>
-    <img src="./docs/NetBox Operator High-Level Architecture.png" alt="Diagram: NetBox Operator High-Level Architecture" width="800"/>
-    <p><em>Figure 1: NetBox Operator High-Level Architecture</em></p>
-</div>
+![Figure 1: NetBox Operator High-Level Architecture](docs/netbox-operator-high-level-architecture.drawio.svg)
```

Comment:
This looks odd with github in dark mode - maybe you could add a white background? also I think that there is a missing icon for k8s

<img width="1101" alt="image" src="https://github.com/user-attachments/assets/e9117ed1-fbd7-4902-86eb-c4978cd1a178">